src/components/AccountModule/connection-hooks.js

```diff
@@ -71,8 +41,7 @@ export function useWalletConnectionDetails(
     isWalletAndClientSynced
 
   const defaultOkConnectionDetails = {
-    connectionMessage: `Connected to ${walletNetworkName}`,
-    connectionMessageLong: `Connected to Ethereum ${walletNetworkName} Network`,
```

Comment:
We weren't using this key as far as I could tell

Reply:
Perhaps the previous implementation wasn't correct but I think there does need to be a distinction between a long and short message.

Previously
![image](https://user-images.githubusercontent.com/11708259/87296359-8af0de00-c4fe-11ea-9619-54124b17396d.png)

This change
![image](https://user-images.githubusercontent.com/11708259/87296335-80cedf80-c4fe-11ea-8559-e685af3086fe.png)

The previous version with the button length being shorter than the pop over is a tighter presentation choice imo.

Reply:
👍 to this; we definitely want shorter messages for the short connection message.